src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -332,13 +335,110 @@ public override IntermediateNode Visit(TexlCallNode node, IRTranslatorContext co
                     }
                 }
 
+                args = ReplaceBlankWithDefault(args, func);
+
                 // this can rewrite the entire call node to any intermediate node.
                 // e.g. For Boolean(true), Instead of IR as Call(Boolean, true) it can be rewritten directly to emit true.
                 var irNode = func.CreateIRCallNode(node, context, args, scope);
 
                 return MaybeInjectCoercion(node, irNode, context);
             }
 
+            private List<IntermediateNode> ReplaceBlankWithDefault(List<IntermediateNode> args, TexlFunction func)
+            {
+                var len = args.Count;
+                List<IntermediateNode> convertedArgs = new List<IntermediateNode>(len);
+
+                for (var i = 0; i < len; i++)
+                {
+                    IntermediateNode convertedNode = default;
```

Comment:
>default [](http://example.com/codeflow?start=53&length=7)

nit - since we're always assigning this, we can actually leave it blank here. 
Then the compiler will verify all paths set an assignment. 

 IntermediateNode convertedNode;